hack/verify-file-perms.sh

```diff
@@ -17,7 +17,19 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-files=$(find . -name "*.sh" \! -perm /a+x)
+# BSD sed doesn't have --version, and needs + instead of /
+# GNU sed deprecated and removed +
+desired_perm="/111"
+# we're not actually running a search so SC2185 doesn't apply
+# shellcheck disable=SC2185
```

Comment:
Can't we inline this with the if so it just disables that line rather than everything after it?

Reply:
this doesn't disable the whole file, to disable the whole file you have to put it directly after the shebang https://github.com/koalaman/shellcheck/wiki/Ignore#ignoring-all-instances-in-a-file-044

Reply:
except after a shebang, or as a CLI flag, disables are specific to the following line